reproduce-figures/prepare-scpca-portal-data.R

```diff
@@ -266,6 +277,20 @@ input_zips |>
             }
 
       })
+      
+      # Save metadata files to use when preparing the sample and library metadata files later
+      project_bulk_tsv <- file.path(project_scratch_dir, glue::glue("{project_id}_bulk_metadata.tsv"))
+      if (file.exists(project_bulk_tsv)) {
+        fs::file_copy(project_bulk_tsv, bulk_metadata_scratch_dir, overwrite = TRUE)
+      }
+      
+      if (project_id %in% citeseq_projects) {
+        fs::file_copy(
+          file.path(project_scratch_dir, glue::glue("single_cell_metadata.tsv")), 
+          file.path(citeseq_metadata_scratch_dir, glue::glue("{project_id}_single_cell_metadata.tsv")), 
+          overwrite = TRUE
+        )
+      }
```

Comment:
Note some of my comments below, but I think you could remove this entirely and you don't need to copy metadata files from one scratch to the next scratch.